### PR TITLE
Collect the experiment results into csv file

### PR DESCRIPTION
**Pull request by feliciahalim**
.. By adding a new target in the coreutils Makefile.

### CODE REVIEW

File: coreutils/Makefile [hunk 1]
```diff
@@ -410,3 +410,48 @@ exp-201609:
  		make $$program.tracerx3; \
 		make $$program.tracerx4; \
 	done
+
+# This target is used for collecting data from experiment exp-201609 result into log-exp-201609.csv file.
+csv-exp-201609:
+	rm -f log-exp-201609.csv
+	echo "Directory,Time, #instructions,#completed paths,#error paths,#subsumed paths,Average branching depth of completed paths,Average branching depth of subsumption paths,Average instructions of completed paths, Average instructions of subsumed paths,Icov,Bcov,Gcov,SLOC" >> log-exp-201609.csv; \
```
Comment (domainexpert) [inline coreutils/Makefile]: I think we also need:
1. `#program exit paths`. I know that this number should equal `#completed paths` - `#error paths` - `#subsumed paths`, but if there were anomalies, we would like to detect it.
2. Time for actual solver calls in subsumption check (ms).
3. Number of solver calls for subsumption check.
4. Number of solver calls for subsumption check that resulted in subsumption failure.
5. Average table entries per subsumption checkpoint.
6. Average solver calls per subsumption check.

File: coreutils/Makefile [hunk 2]
```diff
@@ -410,3 +410,48 @@ exp-201609:
...
+	for program in cut expand fmt fold join pr printf sort sum tail tsort uniq wc factor seq; \
```
Comment (domainexpert) [inline coreutils/Makefile]: Here the loop that iterates over `ext` should be the outer one, and the loop that iterates over `program` the inner loop. The experimental table will be easier to read with if the runs are first categorized according to the tool first before the individual problem.

**Review by domainexpert — CHANGES_REQUESTED** [review]
Some changes.

**Comment by feliciahalim** [comment]
@domainexpert Thank you for the suggestion. I have added more data to be printed in the CSV file.

**Review by domainexpert — APPROVED** [review]

**Comment by domainexpert** [comment]
LGTM.

**Pull request merged**